### DISCUSSION TITLE
Trace functionality not shown bridge or wireless link

### DISCUSSION THREAD

**Discussion by Abulhallaj** [netbox-community/netbox#8219] [Q&A]
As a continue to issue 8209
[https://github.com/netbox-community/netbox/issues/8209](https://github.com/netbox-community/netbox/issues/8209)

NetBox version
v3.1.3

Python version
3.8

### Steps to Reproduce
We have some wireless radio devices that connect sites. in every site the LAN interfaces on this devices is connected to Switch and documented, also WLAN interfaces are connected together via wireless link. In every device we create a bridge1 interface and bridged the WLAN and LAN interface into it.

### Expected Behavior
```
------------------
|                |
|      SW01      |
|                |
-----| LAN |------
        |
        | Cable
        |
-----| LAN |------
|                |  \
|WirelessDevice01|   bridge1
|                |  /
-----| WLAN |-----
        |
        | Wireless Link
        |
-----| WLAN |-----
|                |  \
|WirelessDevice02|   bridge1
|                |  /
-----| LAN |------
        |
        | Cable
        |
-----| LAN |------
|                |
|      SW02      |
|                |
------------------
```

### Observed Behavior
The Cable trace just shown the Cable path and not the bridge area or wireless link was shown in trace as excepted. wireless link can traced exclusively but just shown wireless link.

@jeremystretch 
> This is expected behavior. Any number of interfaces can be attached to a bridge group, so the cable trace cannot assume the next hop. If you'd like to enable tracing directly across these, you can bridge the LAN and WLAN interfaces to one another directly.

I also do that and bridge interfaces together but that's not work

![bridged interfaces ](https://user-images.githubusercontent.com/20490198/148011872-984dcc73-d684-471d-afe1-fb28001fd259.png)

![LAN trace](https://user-images.githubusercontent.com/20490198/148012025-a278fa5d-5929-4375-8fe8-f09c17a83351.png)

![WLAN trace](https://user-images.githubusercontent.com/20490198/148012065-874e94fa-18c1-45fc-876b-e58c45d5ae39.png)

**Comment by Abulhallaj** [comment]
Still need help! if it possible please add some expample to [https://demo.netbox.dev](https://demo.netbox.dev)
@jeremystretch

**Reply by jeremystretch** [reply]
Please stop pinging me. I'm not the only one who can help.
